CHANGELOG.md

```diff
@@ -1,21 +1,25 @@
-# 0.32 (in development)
+# 0.32, March 2018
 
 New features:
 
-* ...
-
-Bug fixes:
-
-* ...
+* A new embedded configuration is used when TruffleRuby is used from another
+language or application. This disables features like signals which may
+conflict with the embedding application such as threads, and enables features
```

Comment:
like ... with ...
This disables features like signals and threads which may conflict with the embedding application, and ...

Reply:
Fixed.